gen/groundobjectsgen.py

```diff
@@ -589,20 +589,43 @@ def __init__(
         self.tacan_registry = tacan_registry
 
     def generate(self) -> None:
+
+        # Note : Helipad are generated as neutral object in order not to interfer with capture triggers
+        neutral_country = self.m.country(self.game.neutral_country.name)
         country = self.m.country(self.game.coalition_for(self.cp.captured).country_name)
         for i, helipad in enumerate(self.cp.helipads):
             name = self.cp.name + "_helipad_" + str(i)
             logging.info("Generating helipad : " + name)
-            pad = SingleHeliPad(name=(name + "_unit"))
+            pad = InvisibleFARP(name=name)
```

Comment:
Why the change? `SingleHeliPad` is the proper marked helipad, but `InvisibleFARP` is just an arbitrary ground location, right?

Reply:
Yes, I encountered somes issues with the Mi-24 not spawning properly on the "SingleHelipad". It would spawn broken, like if it was in the ground and collided with the static object. The issue does not happen with the InvisibleFARP one, so i went for it instead. The landing area is still recognizable for landing as there is the small FARP ammo and fuel depot just in front of the spot.

Reply:
Ah, gotcha. Add a comment so someone else doesn't come along and think that would be a harmless change?